### DISCUSSION TITLE
API tokens

### DISCUSSION THREAD

**Discussion by alexandrepletty** [directus/directus#13887] [Archive]
Is it possible to add the creation of an api token in the settings?
This would secure data transactions between directus and the web service without the user being logged in or using a static token

**Comment by rijkvanzanten** [comment]
How would this be different from using a static token? Are you imagining that this token you generate in settings is linked to a role rather than a user?

**Reply by alexandrepletty** [reply]
It would be easier to have a section where the API tokens are grouped in the parameters and we could add a name, description but above all a type, which makes it possible to limit access (read only, total access, etc.) would be more direct and the token would be limited to the application rather the user.
